docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
 [[introduction]]
-== Introduction
+== {kib} &mdash; your window into the Elastic Stack
+++++
+<titleabbrev>What is Kibana?</titleabbrev>
+++++
 
-Kibana is an open source analytics and visualization platform designed to work with Elasticsearch. You use Kibana to
-search, view, and interact with data stored in Elasticsearch indices. You can easily perform advanced data analysis
-and visualize your data in a variety of charts, tables, and maps.
+**_Explore and visualize your data and manage all things Elastic Stack._**
 
-Kibana makes it easy to understand large volumes of data. Its simple, browser-based interface enables you to quickly
-create and share dynamic dashboards that display changes to Elasticsearch queries in real time.
+Whether you’re a user or admin, {kib} makes your data actionable by providing
+three key functions:
 
-Setting up Kibana is a snap. You can install Kibana and start exploring your Elasticsearch indices in minutes -- no
-code, no additional infrastructure required.
+* **{kib} is an open-source analytics and visualization platform.**
+Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+
+* **{kib} is a UI for managing {es} clusters.**
+Manage your security settings, assign user roles, monitor the Stack,
+ingest and roll up your data...the list goes on.
+
+* **{kib} is a centralized hub for our solutions.** &mdash; From log analytics to
+document discovery to SIEM, {kib} is the portal for accessing these and other capabilities.
+
+[role="screenshot"]
+image::images/intro-kibana.png[]
+
+[float]
+[[where-to-start-kibana]]
+=== Where do I start?
+
+There is no faster way to try out {kib} than with our hosted {es} Service.
+https://www.elastic.co/cloud/elasticsearch-service/signup[Sign up for a free trial]
+and start exploring data in minutes.
+
+You can also install {kib} on your own &mdash; no code, no additional
+infrastructure required. If you don’t have {es}, you’ll need that too.
+Think of Elasticsearch as the engine that stores and processes the data, with {kib} sitting on top.
+
+[float]
+[[get-data-into-kibana]]
+=== Try {kib} with our data sets and tutorials
+
+Our <<add-sample-data, sample data sets>> enable you to take {kib} for a test ride without having
+to go through the process of loading data yourself. Or, try our <<getting-started, built-in tutorials>>,
+which can help you quickly set up data flows between your app and the Elastic Stack.
+
+[role="screenshot"]
+image::images/intro-data-tutorial.png[]
+
+With both scenarios, you get data, index patterns, visualizations, and dashboards.
+<<index-patterns, Index patterns>> are key to Kibana &mdash; they identify which
+{es} indices contain the data that you want to work with.
+
+If a tutorial doesn’t exist for your data, go to the
+{beats-ref}/beats-reference.html[Beats overview] to learn about other data shippers
+in the {beats} family.
+
+[float]
+[[explore-and-query]]
+=== Explore & query
```

Comment:
I like the consistency of `Explore & query`, `Visualize & analyze`, and `Organize & secure`, but the other section titles use `and`. Maybe make this consistent across all of these titles?